resources/RegexGrammar.pp

```diff
@@ -57,8 +59,8 @@
 %token class:literal             \\.|.|\n
 
 // Internal options.
-// See https://www.regular-expressions.info/refmodifiers.html
-%token  internal_option          \(\?([imsxnJUX^]|xx)?-?([imsxnJUX^]|xx)\)
+// See https://www.php.net/manual/en/regexp.reference.internal-options.php
```

Comment:
I find the other document better as it contains for example `(?^)` which is not in the php docs. Please keep both links at least.

Reply:
done